docs/gambit/mutation-verifier.md

```diff
@@ -36,96 +36,80 @@ Once you have updated your `certora-cli` installation using `pip` to get the rel
 dependencies, run Gambit from the command line:
 
 ```
-certoraMutate path/to/config/file/Example.conf
+certoraMutate --prover_conf path/to/prover/prover.conf --gambit_conf path/to/gambit/gambit.conf
 ```
 
-(gambit-prover-config)=
-## Configuration
-The tool expects a configuration file (extension `.conf` is required).
-which is a JSON object.
-Here is an example configuration file:
+## Configurations
+The tool expects two separate configuration files (extension `.conf` is required for both):
```

Comment:
I'm not sure we care about the actual extension

Reply:
agreed this is dated